content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+fn vertex(vertex: Vertex) -> VertexOutput {
+    let world_position = vec4<f32>(vertex.position, 1.0);
+    var out: VertexOutput;
+    // The `view` variable comes from the imported bind group 
+    out.clip_position = view.view_proj * world_position;
+    return out;
+}
+```
+
+We also plan to experiment with using Naga for "partial imports" of specific, named symbols (ex: import a specific function or struct from a file).
+
+### Pipeline Specialization
+
+<div class="release-feature-authors">authors: @cart</div>
+
+When shaders use a preprocessor and have multiple permutations, the associated "render pipeline" needs to be updated to accommodate those permutations (ex: different Vertex Attributes, Bind Groups, etc). To make this process straightforward, we added the SpecializedPipeline trait, which allows defining specializations for a given key:
+
+```rust
+impl SpecializedPipeline for MyPipeline {
+    type Key = MyPipelineKey;
+    fn specialize(&self, key: Self::Key) -> RenderPipelineDescriptor {
+        // define a specialized pipeline for the given key here
+    }
+}
+```
+
+Implementors of this trait can then easily and cheaply access specialized pipeline variants (with automatic per-key caching and hot-reloading). If this feels too abstract / advanced, don't worry! This is a "mid level power-user tool", not something most Bevy App developers need to contend with.
+
+### Simpler Shader Stack
+
+Bevy now uses [Naga](https://github.com/gfx-rs/naga) for all of its shader needs. As a result, we were able to remove all of our complicated non-rust shader dependencies: `glsl_to_spriv`, `shaderc`, and `spirv_reflect`. `glsl_to_spriv` was a major producer of platform-specific build dependencies and bugs, so this is a huge win!
+
+### Features Ported to the New Renderer
+
+Render logic for internal Bevy crates had to be rewritten in a number of cases to take advantage of the new renderer. The following people helped with this effort:
+
+* bevy_sprites: @cart, @StarArawn 
+* bevy_pbr: Rob Swain (@superdump), @aevyrie, @cart, @zicklag, @jakobhellermann
+* bevy_ui: @Davier
+* bevy_text: @Davier
+* bevy_gltf: Rob Swain (@superdump)
+
+### WebGL2 Support
+
+<div class="release-feature-authors">authors: @zicklag, @mrk-its, @mockersf, Rob Swain (@superdump)</div>
+
+Bevy now has built-in support for deploying to the web using WebGL2 / WASM, thanks to @zicklag adding a native WebGL2 backend to wgpu. There is now no need for the third party `bevy_webgl2` plugin. Any Bevy app can be deployed to the web by running the following commands:
+
+```rust
+cargo build --target wasm32-unknown-unknown
+wasm-bindgen --out-dir OUTPUT_DIR --target web TARGET_DIR
+```
+
+The New Bevy Renderer developers had to carefully operate within the limits of WebGL2 (ex: storage buffers and compute shaders aren't supported in WebGL2), but the results were worth it! You can try out Bevy's WASM support in your browser using our new [Bevy Examples](/examples) page.
```

Comment:
IMO this is a stunning and engaging showcase, and should also be linked at the very top of the release notes.